engines/private/private.cpp

```diff
@@ -964,7 +964,7 @@ void PrivateEngine::drawScreen() {
 		const Graphics::Surface *frame = _videoDecoder->decodeNextFrame();
 		Graphics::Surface *cframe = frame->convertTo(_pixelFormat, _videoDecoder->getPalette());
 		Common::Point center((_screenW - _videoDecoder->getWidth())/2, (_screenH - _videoDecoder->getHeight())/2);
-		surface->transBlitFrom(*cframe, center);
+		surface->blitFrom(*cframe, center);
```

Comment:
Why this was changed?

Reply:
`transBlitFrom` uses a default transparency colour of 0, which causes graphical errors when using a pixel format without any alpha bits.

Reply:
Right, this is unnecessary in during videos.